src/main/java/me/eccentric_nz/TARDIS/commands/tardis/TARDISVersionCommand.java

```diff
@@ -33,13 +38,27 @@
     }
 
     boolean displayVersion(CommandSender sender) {
-        String version = plugin.getPM().getPlugin("TARDIS").getDescription().getVersion();
-        String chunkversion = plugin.getPM().getPlugin("TARDISChunkGenerator").getDescription().getVersion();
+        final PluginManager pm = plugin.getPM();
+        List<String> hooks = plugin.getDescription().getSoftDepend();
+
+        String tardisversion = pm.getPlugin("TARDIS").getDescription().getVersion();
+        String chunkversion = pm.getPlugin("TARDISChunkGenerator").getDescription().getVersion();
         String cb = Bukkit.getVersion();
         String bv = Bukkit.getBukkitVersion();
-        sender.sendMessage(plugin.getPluginName() + "TARDIS version: " + ChatColor.AQUA + version);// + ChatColor.RESET + " running " + implementation + cb);
-        sender.sendMessage(plugin.getPluginName() + "TARDISChunkGenerator version: " + ChatColor.AQUA + chunkversion);
         sender.sendMessage(plugin.getPluginName() + "Server version: " + ChatColor.AQUA + bv + " " + cb);
+        sender.sendMessage(plugin.getPluginName() + "TARDIS version: " + ChatColor.AQUA + tardisversion);
+        sender.sendMessage(plugin.getPluginName() + "TARDISChunkGenerator version: " + ChatColor.AQUA + chunkversion);
+
+        for (Plugin hook : pm.getPlugins()) {
+            final PluginDescriptionFile desc = hook.getDescription();
```

Comment:
Unnecessary final?

Reply:
I mean, the value never "changes", but I guess we don't *need* it